frappe/public/js/frappe/ui/toolbar/energy_points_notifications.js

```diff
@@ -46,11 +46,24 @@ frappe.ui.EnergyPointsNotifications = class {
 				this.$dropdown_list.find('.recent-points-item').last().remove();
 				this.dropdown_items.pop();
 			}
-			let new_item_html = this.get_dropdown_item_html(new_item);
-			$(new_item_html).insertAfter(this.$dropdown_list.find('.points-date-range').eq(0));
+			this.insert_into_dropdown();
 		});
 	}
 
+	insert_into_dropdown() {
+		let new_item = this.dropdown_items[0];
+		let new_item_html = this.get_dropdown_item_html(new_item);
+		let new_item_date_range = this.get_energy_points_date_range(new_item.creation);
+		let current_date_range = this.get_energy_points_date_range(this.dropdown_items[1].creation);
+		if (current_date_range !== new_item_date_range) {
+			let $date_range = $(`<li class="points-date-range text-muted">${__(new_item_date_range)}</li>`);
```

Comment:
Translating here is not useful... [docs](https://frappe.io/docs/user/en/translations)

Translate [here](https://github.com/frappe/frappe/pull/8246/files#diff-9740338ae2b7c576d5fba79f1f745398R76) instead.